client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -215,15 +229,20 @@ const CellText = React.memo(
         label = "?";
         break;
     }
+    const supportClassName = getSupportClassName(
+      support,
```

Comment:
This may at first look like it changes currently existing functionality, but it doesn't. `getSupportClassName` already does `getFirst(support)` to get the properties used for all the pre-existing checks, so passing it `currentSupport`, defined as `getFirst(support)`, was redundant. However, the new check requires I pass the whole support array, so this change is needed.